pkg/api/v1/types.go

```diff
@@ -20,6 +20,7 @@ import (
 	"k8s.io/client-go/pkg/types"
 	"kubevirt.io/kubevirt/pkg/api"
 	"kubevirt.io/kubevirt/pkg/precond"
+	"kubevirt.io/kubevirt/pkg/util"
```

Comment:
Right :+1: 

I will rebase and merge this after #56, then we don't need the content of this "util" package anymore ...